modules/distr-tracing-tempo-config-storage.adoc

```diff
@@ -11,23 +11,35 @@ You can configure object storage for the {TempoShortName} in the `TempoStack` cu
 
 .General storage parameters used by the {TempoOperator} to define distributed tracing storage
 [options="header"]
-[cols="a, a, a, a"]
+[cols="a, a, a"]
 |===
-|Parameter |Description |Values |Default value
-|`spec.storage.secret.type`
-|Type of storage to use for the deployment.
-|`memory`. Memory storage is only appropriate for development, testing, demonstrations, and proof of concept environments because the data does not persist when the pod is shut down.
-|`memory`
-
-|`storage.secretname`
-|Name of the secret that contains the credentials for the set object storage type.
+|Parameter |Description |Values
+
+|`storage.secret.type`
+|Type of object storage for the deployment.
+|`azure`, `gcs`, `s3`
+
+
+|`storage.secret.name`
+|Name of the secret that contains the credentials for the chosen type of object storage.
+|
+
+|`storage.tls.enable`
+|Optional: Enables TLS when communicating with object storage.
 |
-|N/A
 
 |`storage.tls.caName`
-|CA is the name of a `ConfigMap` object containing a CA certificate.
+|Optional: Name of a config map that contains a `service-ca.crt` CA certificate. You must create the config map in the same project as the `TempoStack` custom resource.
 |
+
+|`storage.tls.certName`
+|Optional: Name of a secret that contains a `tls.crt` certificate and `tls.key` private key. You must create the secret in the same project as the `TempoStack` custom resource.
+|
+
+|`storage.tls.minVersion`
+|Optional: The oldest acceptable TLS version.
```

Comment:
Is "SSL/TLS" appropriate here? See guidance: https://redhat-documentation.github.io/supplementary-style-guide/#ssl-tls

Maybe it isn't. Just wondering.

Reply:
Thank you for the suggestion. This is a new product, so use of SSL is very unlikely.